Tasks/Maksim/Task3.1/query07.sql

```diff
@@ -0,0 +1,17 @@
+--Query  7
+SELECT
+    customer.CustomerID,
+    YEAR(SOH.OrderDate) Year,
+    SOH.SalesOrderID,
+    SOH.TotalDue,
+    SUM(SOH.TotalDue) OVER(PARTITION BY YEAR(SOH.OrderDate) ) customer_Sales_Over_Year,
```

Comment:
while it customer sales then it would need to be calculated per customer per year (probably task desc was not clear so no worries :) )